Source/Atomic/UI/UI.h

```diff
@@ -100,6 +100,8 @@ class UI : public Object, private tb::TBWidgetListener
     void ShowDebugHud(bool value);
     void ToggleDebugHud();
 
+    void CycleDebugHudMode();
```

Comment:
Ah right, I've been adding comments or not based on surrounding code. I'll fix this one and keep it mind for future PRs :)